### PR TITLE
ptz controls if broadcast type is ipCamera

### PR DESCRIPTION
**Pull request by lastpeony**
https://github.com/ant-media/Ant-Media-Server/issues/6153

### CODE REVIEW

File: src/web_player.js [hunk 1]
```diff
@@ -1018,6 +1029,8 @@ export class WebPlayer {
         this.setPlayerVisible(false);
 
         this.containerElement.innerHTML = this.videoHTMLContent;
+        this.isIpCameraBroadcast()
```
Comment (mekya) [inline src/web_player.js:1032]: No need to call this method if `restJwt` is blank

**Review by mekya — CHANGES_REQUESTED** [review]
Thank you @lastpeony for the implementation. 

Even if I've not tested, it seems like a good work. Congratz 👏 

I only request to change something in the comment. 

After that I'll test it on my end.

File: src/web_player.js (hunk 1 above)
Comment (lastpeony) [reply src/web_player.js]: Done
Thanks for the review

**Comment by mekya** [comment]
Hi @lastpeony,
In my tests and integration with web panel, I need to add/edit somethings. Please review the changes and let me know if you have any question

Cheers

**Pull request merged**